samtranslator/swagger/swagger.py

```diff
@@ -257,7 +258,8 @@ def add_cors(self, path, allowed_origins, allowed_headers=None, allowed_methods=
                                                                                            allow_credentials)
 
     def add_binary_media_types(self, binary_media_types):
-        self._doc[self._X_APIGW_BINARY_MEDIA_TYPES] = binary_media_types
+        bmt = json.loads(json.dumps(binary_media_types).replace('~1', '/'))
```

Comment:
Dump it into json so all nested lists and dictionaries are expressed as a string, replace the `~1` with `/`, and then load back into data form.